### DISCUSSION TITLE
Outdated documentation vs /docs/ files.

### DISCUSSION THREAD

**Discussion by vmariano** [TanStack/query#4385] [General] [answered]
I was struggling with doc page about dev tools on: https://react-query-v3.tanstack.com/devtools
And notice the import, and instructions and implementation was incorrect:

```
Import the Devtools

The devtools are bundle split into the react-query/devtools package. No need to install anything extra, just:

 import { ReactQueryDevtools } from 'react-query/devtools'
```

However the information on `main` is updated: https://github.com/TanStack/query/blob/main/docs/devtools.md

```
install and Import the Devtools

The devtools are a separate package that you need to install:

$ npm i @tanstack/react-query-devtools
# or
$ pnpm add @tanstack/react-query-devtools
# or
$ yarn add @tanstack/react-query-devtools

You can import the devtools like this:

import { ReactQueryDevtools } from '@tanstack/react-query-devtools'
```

Could be a issue on publish changes?

**Comment by TkDodo** [comment, marked as answer]
v3 docs are for v3, TanStack docs are for v4. devtools are imported differently between the two versions. The docs are correct.

**Reply by vmariano** [reply]
Sorry, I didn't notice there was targeting another version. It might be nice, have a link mentioning that on https://react-query-v3.tanstack.com since is the 2nd google result.
